lib/repositories/firehose-repository.ts

```diff
@@ -0,0 +1,53 @@
+import { Firehose } from 'aws-sdk';
+import { AnalyticsEvent, AnalyticsEventType } from '../entities/analytics-events';
+import { default as Logger } from 'bunyan';
+
+export class FirehoseLogger {
```

Comment:
Might be nice to make this implement an interface like `IAnalyticsLogger` or something. Then we have a nice interface if we ever swapped from using firehose. We have a similar pattern [here](https://github.com/Uniswap/uniswapx-parameterization-api/blob/88db0607dea1ff620db4ae612ac819f1369a74de/lib/providers/webhook/s3.ts#L10).